shared/src/commonMain/kotlin/com/intive/picover/common/result/ResultContractLauncher.kt

```diff
@@ -3,8 +3,10 @@ package com.intive.picover.common.result
 import androidx.compose.runtime.Composable
 import dev.gitlive.firebase.storage.File
 
-expect class ResultContractLauncher {
-	fun launch()
+class ResultContractLauncher(private val onLaunch: () -> Unit) {
```

Comment:
Now it's more flexible and will be used for firebase auth ui contract in upcoming PR

Reply:
I'm thinking if maybe it would be simpler to just use lambda instead of this additional layer of wrapping into `class`? Or an alias for lambda? Something to consider 😄